metrics/metrics.go

```diff
@@ -0,0 +1,135 @@
+package metrics
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-io/go-utils/v2/pathutil"
+)
+
+const defaultEndpoint = "gradle-analytics.services.bitrise.io"
+const defaultPort = 443
+
+// Sync the major version of this step and the plugin.
+// Use the latest 1.x version of the plugin, so we don't have to update this definition after every plugin release.
+// But don't forget to update this to `2.+` if the library reaches version 2.0!
+const defaultPluginVersion = "main-SNAPSHOT" // TODO: change to 1.+
```

Comment:
There is a todo here and in the go template file. Do we (the mobile team) need to have this in mind whenever we work on this step or this will be managed by someone from the ACI team?